README.md

```diff
@@ -1,186 +1,80 @@
-
-    ```plaintext
-    INFO: Success! Private key written to ./mockSpacestationPrivateKey. Run these commands to SSH into your machines...
-    ssh -i mockSpacestationPrivateKey azureuser@mockgroundstation-abcd1234efgh5.eastus.cloudapp.azure.com
-    ssh -i mockSpacestationPrivateKey azureuser@mockspacestation-abcd1234efgh5.australiaeast.cloudapp.azure.com
-    ```
-
-## Synchronize Directories
-
-Once you're connected to the Spacestation, **any files or directories that make their way to the `/home/azureuser/trials` directory will be synched to the same directory on the Groundstation at a rate of 2 megabits per second every minute**.
-
-This scheduled synchronization recreates the time delay and limited bandwidth environment of a real-world experiment executed on the ISS.
-
-![The Mock Groundstation and Mock Spacestation /trials directories in sync](docs/images/synched-stations.png)
-
-1. Place a file or directory in `/home/azureuser/trials`:
-
-    ```bash
-    # on the Mock Spacestation
-    echo "Hello! It is currently $(date) on the mockSpacestation! Happy Hacking!" >> /home/azureuser/trials/hello.txt
-    ```
-
-2. And within a minute or so, on the Mock Groundstation, you should see that file in the same directory:
-
-    ```bash
-    # now on the Mock Groundstation
-    cd /home/azureuser/trials
-    cat hello.txt
-    Hello! It is currently Fri Aug 20 21:10:10 UTC 2021 on the mockSpacestation! Happy Hacking!
-    ```
-
-3. On the Mock Spacestation, you can inspect the contents of `azure-sync.log` to see file and directory transmission history and transfer speeds:
-
-    ```bash
-    # back on the Mock Spacestation
-    cat /home/azureuser/azure-sync.log
-    ```
-
-4. Which yields output from `rsync` operations like:
-
-    ```plaintext
-    sent 177 bytes  received 66 bytes  44.18 bytes/sec
-    total size is 92  speedup is 0.38
-    opening connection using: ssh -i /home/azureuser/.ssh/mockSpacestationPrivateKey -l azureuser mockgroundstation-abcd1234efgh5.eastus.cloudapp.azure.com rsync --server -vvlogDtprze.iLsfxC --bwlimit=250 . /home/azureuser/trials  (12 args)
-    sending incremental file list
-    delta-transmission enabled
-    hello.txt is uptodate
-    total: matches=0  hash_hits=0  false_alarms=0 data=0
-    ```
-
-**Happy hacking!** Continue reading on for more information about how we built the Genomics experiment on Azure using the HPE SBC2 and the ISS, or how we setup our developer machines with containers to collaborate.
-
-## An Example "Burst Down" Workload
+# Quick Start to Azure
+1. [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fmock-spacestation%2Fmain%2FAzureVM.json)
+1. SSH into new Ground Station VM (check Output from Template for quick copy/paste SSH command)
```

Comment:
for a lot of users the Portal deployment output is an unused feature, let's include a screenshot where they can grab the SSH command